login.lp

```diff
@@ -43,21 +43,20 @@ mg.include('scripts/pi-hole/lua/header.lp','r')
                 <form id="loginform">
                     <div class="login-options has-feedback" id="pw-field">
                         <input type="text" id="username" value="pi.hole" autocomplete="username" hidden>
-                        <div class="pwd-field form-group">
+                        <div class="input-group pwd-field form-group">
                             <!-- hidden username input field to help password managers to autfill the password -->
-                            <input type="password" id="loginpw" class="form-control" placeholder="Password" value="" spellcheck="false" autocomplete="current-password" autofocus>
-                            <span class="fa fa-key pwd-field form-control-feedback"></span>
-                        </div>
-                        <div class="form-group hidden" id="totp_input">
-                            <input type="text" id="totp" size="6" maxlen="6" class="form-control totp_token" placeholder="" value="" spellcheck="false" autocomplete="current-password" autofocus>
-                            <span class="fa-solid fa-clock-rotate-left pwd-field form-control-feedback"></span>
+                            <input type="password" class="form-control" placeholder="Password" value="" spellcheck="false" autocomplete="current-password" id="current-password" autofocus required>
+                            <span class="input-group-btn">
+                                <button class="btn btn-default" id="toggle-password" type="button" title="Show password as plain text. Warning: this will display your password on the screen.">
+                                    <span class="fa fa-fw fa-eye field-icon"></span>
+                                </button>
+                            </span>
                         </div>
                     </div>
-                    <!--
-                    <div class="form-group" title="Pi-hole has to set a cookie for the login session to be successful. The cookie will not contain your password nor is it used anywhere outside of you local Pi-hole.">
-                        <input type="checkbox" id="logincookie" checked>
-                        <label for="logincookie">Keep me logged in (uses cookie)</label>
-                    </div> -->
+                    <div class="form-group has-feedback hidden" id="totp_input">
+                        <input type="text" id="totp" size="6" maxlen="6" class="form-control totp_token" placeholder="123456" value="" spellcheck="false" autofocus>
```

Comment:
```suggestion
                        <input type="text" id="totp" size="6" maxlen="6" class="form-control totp_token" placeholder="123456" value="" spellcheck="false" autocomplete="off" autofocus>
```

Not sure if this is the right command, but there should be no (sensless) autofill/complete on the TOTP input field